source/WHIMChannel.mc

```diff
@@ -112,6 +112,15 @@ class WHIMChannel extends Ant.GenericChannel
                             System.println( "Response event: EVENT_TX" );
                             break;
 
+                        case Ant.MSG_CODE_EVENT_TRANSFER_TX_COMPLETED:
+                            System.println("Response event: TRANSFER_TX_COMPLETED");
+                            break;
+
+                        case Ant.MSG_CODE_EVENT_TRANSFER_TX_FAILED:
+                            System.println("Response event: TRANSFER_TX_FAILED");
+                            sendResetDataCommand();
```

Comment:
Are we sure we want to re-send the command here?  I feel like this may be assuming a lot about the operation of the sensor.  If Nick is certain this is OK, then I will approve it but I feel like it may be better to take a different approach to resending the command.

Reply:
In a conversation outside of this thread, Nick and I agreed this was fine as we are only sending one data page for now.